hcsctl/pkg/rook/rook.go

```diff
@@ -25,27 +25,44 @@ const (
 )
 
 var (
-	// PriorityYaml represents priority.yaml
+	// SnapshotCrdsYaml represents snapshot_crds.yaml
+	SnapshotCrdsYaml string = "snapshot_crds.yaml"
+
+	// SnapshotControllerRbacYaml represents snapshot-controller-rbac.yaml
+	SnapshotControllerRbacYaml = "snapshot-controller-rbac.yaml"
+	// SnapshotControllerYaml represents snapshot-controller.yaml
+	SnapshotControllerYaml string = "snapshot-controller.yaml"
+	
+        // PriorityYaml represents priority.yaml
```

Comment:
Please run lint check on local. It seems like tab setting is misconfigured. You can do either `make -C hcsctl check` or `golangci-lint run`. Unnecessary empty spaces in following lines should be removed on this file. (35, 63, 64, 87, 144, 361, 417)

Reply:
Reflected. Thank you